aks/terraform.tfvars.json

```diff
@@ -1,15 +1,15 @@
 {
   "location": "xxxxxxxxxxxxxxxxxxxxxxxxx",
-  "az_resource_group": "xxxxxxxxxxxxxxxxxxxxxxxxx",
+  "resource_group": "xxxxxxxxxxxxxxxxxxxxxxxxx",
   "ssh_public_key": "/xxxxxxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxxxx.pub",
-  "agent_admin": "xxxxxxxxxxxxxxxxxxxxxxxxx",
+  "ssh_username": "xxxxxxxxxxxxxxxxxxxxxxxxx",
```

Comment:
I assume the ssh public key is not being sourced from the file-system and the actual contents of the public key are expected here?

Reply:
This file is cruft; removed in 1e04545 .